docs/uma-tokenholders/adding-price-id.md

```diff
@@ -71,11 +71,11 @@ To view the list of approved mainnet price identifiers, see below or run the `Su
 
 ### List of approved collateral currencies
 
-|Collateral Currency| Link to UMIP|
-|:-------| :-----------|
-|WETH|  [UMIP-10](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-10.md)
-|renBTC|  [UMIP-11](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-11.md)
-|PERL|  [UMIP-12](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-12.md)
-|DAI| 
-|USDC; USDT|  [UMIP-18](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-18.md)
-|rDAI|  [UMIP-17](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-17.md)
+|Collateral Currency| Final Fee | Link to UMIP|
+|:-------| :-----------| :-----------|
+|WETH| 1 |  [UMIP-10](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-10.md)
+|renBTC| 0.035 |  [UMIP-11](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-11.md)
+|PERL| 8600 | [UMIP-12](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-12.md)
+|DAI| 400 | 
```

Comment:
Should the UMIP be the same as rDAI?

Reply:
Grandfathered in with the introduction of the whitelist. Adding the whitelist umip-8 link